tests/h/search/config_test.py

```diff
@@ -178,12 +195,78 @@ def test_updates_index_atomically(self, client):
 
     def test_raises_if_called_for_concrete_index(self, client):
         """Raise if called for a concrete index."""
-        client.conn.indices.get_alias.side_effect = NotFoundError('test', 'test desc')
+        client.conn.indices.get_alias.side_effect = (
+            elasticsearch1.exceptions.NotFoundError('test', 'test desc'))
 
         with pytest.raises(RuntimeError):
             update_aliased_index(client, 'new-target')
 
 
+class TestUpdateIndexSettings(object):
+    @pytest.mark.parametrize(
+        'esversion,mapping',
+        ((ELASTICSEARCH1_VERSION, ANNOTATION_MAPPING),
+         ('6.2.0', ES6_ANNOTATION_MAPPING))
+    )
+    def test_succesfully_updates_the_index_settings(self, client, esversion, mapping):
+        client.version = esversion
+        client.conn.indices.get_alias.return_value = {
+            'old-target': {'aliases': {'foo': {}}},
+        }
+        client.conn.indices.get_settings.return_value = {
+            "old-target": {
+                "settings": {
+                    "index": {
+                        "analysis": {
+                            "old_setting": "val",
+                        }
+                    }
+                }
+            }
+        }
+        update_index_settings(client)
```

Comment:
Can you add a blank line before and after the `update_index_settings` call to make it clearer which are the _arrange_, _act_ and _assert_ phases of the test please.